src/Checks/SessionCheck.php

```diff
@@ -62,7 +62,7 @@ public function getCookie(ResponseInterface $response)
         $cookies = $response->getHeader('Set-Cookie');
 
         foreach ($cookies as $cookie) {
-            if (strpos($cookie ?? '', 'SESSID') !== false) {
+            if (strpos($cookie ?? '', 'PHPSESSID') !== false) {
```

Comment:
```suggestion
            if ($cookie === 'PHPSESSID') {
```
We're checking for specifically this named cookie now, not any cookie sort-of-like it

Reply:
Keep as is, the value will be something like `PHPSESSID=3nalnh9k5habrdt9blok3jp7d1; path=/; HttpOnly; SameSite=Strict`